database.py

```diff
@@ -170,32 +170,34 @@ def names_with_infix(self, infix):
                 if infix in str(pokemon.get_name())]
 
     def __load_data(self):
-        # Load all the Pokemon data. This does not include the 'Extra' Pokemon.
-        with open(self.directory + "/./Data/pokemon.txt", 'r') as data_file:
-            for line in data_file:  # Load everything but the Pokemon from the 'Extra' folder.
-                identifier, _, name = line.strip().partition(' ')
-                identifier = '{:03}'.format(int(identifier))
-                region = self.__determine_region(identifier)
-                path = self.__determine_folder(identifier) + "/" + identifier + ".jpg"
-                pokemon = Pokemon(identifier, name.lower(), region, path)
+        """Load all the Pokemon data. This does not include the 'Extra' Pokemon."""
+        with open(self.directory + "/./Data/light-dark.txt", 'r') as data_file:
```

Comment:
First, is the first slash necessary? Secondly, this seems off, you are loading all the same data from a completely different file and there are no changes to the files themselves? Are the databases that badly redundant?